internal/distributedtracing/distributedtracing.go

```diff
@@ -99,7 +102,31 @@ func Init(ctx context.Context, raw []byte, id string) (*otlptrace.Exporter, *tra
 		tlsOption,
 	)
 
+	attrs := make([]attribute.KeyValue, len(distributedTracingConfig.ResourceAttributes))
+	for key, value := range distributedTracingConfig.ResourceAttributes {
+		switch v := value.(type) {
+		case string:
+			attrs = append(attrs, attribute.String(key, v))
+		case json.Number:
+			// Convert json.Number to float64 or int as needed
+			num, err := v.Int64()
+			if err == nil {
+				attrs = append(attrs, attribute.Int64(key, num))
+			} else {
+				num, err := v.Float64()
+				if err == nil {
+					attrs = append(attrs, attribute.Float64(key, num))
+				}
+			}
```

Comment:
Should we return an error if we can't convert to float64 or int?

Reply:
sure, done.